src/SayMore/UI/ComponentEditors/SessionBasicEditor.cs

```diff
@@ -619,5 +626,123 @@ private void HandlePeopleEditClick(object sender, EventArgs e)
 				break;
 			}
 		}
+
+		/// ------------------------------------------------------------------------------------
+		/// <summary>
+		/// Method to save the new AccessProtocol node and update the existing AccessProtocol node when option get changed
+		/// </summary>
+		/// <param name="accessCode">Previous AccessCode</param>
+		/// <returns>New AccessCode</returns>
+		/// ------------------------------------------------------------------------------------
+		private string HandleAccessProtocols(string accessCode)
+		{
+			string currentAccessCode = accessCode;
+			if (Program.CurrentProject == null) return currentAccessCode;
+
+			string sessionFile = _file.PathToAnnotatedFile;
+			var accessProtocol = Program.CurrentProject.AccessProtocol;
+
+			var choices = _accessOptions.ToList();
```

Comment:
_accessOptions is already a list. ToList makes a copy of it (with a name that is shorter but more vague). This method does not appear to be making any changes to the list, so there should be no need to make a copy. I'm guessing this line just got copied from SetAccessCodeListAndValue(), which _does_ modify the list, and therefore needs a copy of it.